chpl/chpl-resources/src/main/resources/errors.properties

```diff
@@ -266,6 +266,25 @@ listing.upload.couldNotParse=Could not read uploaded file: "%s".
 listing.chplProductNumber.notUnique=The expected CHPL Product Number %s must be unique among all other certified products but one already exists with this value.
 listing.chplProductNumber.changedNotUnique=The CHPL Product Number has changed. The new CHPL Product Number "%s" must be unique among all other certified products but one already exists with this value.
 
+listing.duplicateTestingLab=Listing contains duplicate Testing Lab: '%s'.  The duplicates have been removed.
```

Comment:
Our other errors messages don't have the `  ` double space after a period. While I know they tend to get ignored in most displays, I'd like to stay consistent

Reply:
I'm sure we were all taught to double-space after a period but that's not how the cool kids do it anymore!